lib/nuget-parser/parsers/dotnet-core-v2-parser.ts

```diff
@@ -153,26 +154,6 @@ function buildGraph(
     return { ...acc, [nameWithVersion]: pkg };
   }, {});
 
-  const topLevelDepPackages = topLevelDependencies.reduce(
-    (acc, topLevelDepName) => {
-      const nameWithVersion = Object.keys(targetDependencies).find(
-        (targetDep) =>
-          // Lowercase the comparison, as .csproj <PackageReference>s are not case-sensitive, and can be written however you like.
-          targetDep.toLowerCase().startsWith(topLevelDepName.toLowerCase()),
-      );
-      if (!nameWithVersion) {
-        throw new InvalidManifestError(
-          `cant find a name and a version in assets file, something's very malformed`,
-        );
-      }
-
-      const [name, version] = nameWithVersion.split('/');
-
-      return { ...acc, [name]: version };
-    },
-    {},
-  );
-
```

Comment:
This makes no sense, not sure what I was doing. Think it's a regression from my very first attempt at generating this.